src/make_parts.zsh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env zsh
+
+rm parts/*
+files=(Actions Content Description Divider Extra Header Item Menu Meta Label Value Avatar Bar Date Event ImageContent Summary Title)
+
+for file in $files; do
```

Comment:
This was the original script that generated full component part files.  This is now done away with by the component part factory above